table/tables/tables.go

```diff
@@ -48,9 +48,8 @@ import (
 // tableCommon is shared by both Table and partition.
 type tableCommon struct {
 	tableID int64
-	// partitionID is a unique int64 to identify a partition, it equals to tableID
-	// if this tableCommon struct is not a partition.
-	partitionID     int64
+	// physicalID is a unique int64 to identify a physical TiKV region.
```

Comment:
What is the difference between this id and region id? Region id also identifies a physical TiKV region. If they are two different concepts, we're better to write the difference down explicitly.

Reply:
This comment is wrong, I'll fix it.